coffee_cmd/src/main.rs

```diff
@@ -26,15 +26,17 @@ async fn run(args: CoffeeArgs, mut coffee: CoffeeManager) -> Result<(), CoffeeEr
             } else {
                 None
             };
-            let result = coffee.install(&plugin, verbose, dynamic).await;
-            if let Some(spinner) = spinner {
-                if result.is_ok() {
-                    spinner.finish();
-                } else {
-                    spinner.failed();
+            match coffee.install(&plugin, verbose, dynamic).await {
+                Ok(_) => {
+                    spinner.and_then(|spinner| Some(spinner.finish()));
+                    if verbose {
+                        term::success!("Plugin {plugin} Compiled and Installed")
+                    }
```

Comment:
```suggestion
```

nit: With the verbose flag we skip just the spinner, because does not make sense inside the terminal state.

Reply:
Then you get no notification of a success in verbose mode ?!

Reply:
>Then you get no notification of a success in verbose mode ?!

The opposite? you get a notification in both cases?

Reply:
I don't get it. Your "Suggested Change" is removing the lines that inform the user in verbose mode. Then the only thing left that indicates success or error is the spinner. But the spinner is only active in non-verbose mode.

Reply:
Ok now it makes the confusion a little clearer (your change has a `-` infront of all 3 lines!). One thing left: my code is not a behavioural change for the `term::success`. Before it was only run in verbose mode aswell. This is because it was only run if `spinner` was `None` and `spinner` was only `None` if `verbose==true`:
```
let spinner = if !verbose {
    Some(...)
} else {
    None
}
if let Some(spinner) = spinner {
    ...
}  else if result.is_ok() {
    term::success!("Plugin {plugin} Compiled and Installed")
}

Reply:
I told you branching is shit with rust.

This was a bug, so please remove the if and print `term::success!("Plugin {plugin} Compiled and Installed")` always